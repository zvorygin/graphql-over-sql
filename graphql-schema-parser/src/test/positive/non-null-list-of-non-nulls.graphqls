schema {
    query: Query
}

scalar Url

type Query {
    url : [Url!]!
}