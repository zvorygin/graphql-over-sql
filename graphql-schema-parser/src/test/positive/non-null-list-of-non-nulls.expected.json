{
  "schema": {
    "queryType": {
      "collection": false,
      "nonNull": false,
      "typeName": "Query"
    }
  },
  "types": {
    "Query": {
      "fields": [
        {
          "name": "url",
          "type": {
            "collection": false,
            "nonNull": true,
            "wrappedType": {
              "collection": true,
              "nonNull": false,
              "wrappedType": {
                "collection": false,
                "nonNull": true,
                "wrappedType": {
                  "collection": false,
                  "nonNull": false,
                  "typeName": "Url"
                }
              }
            }
          }
        }
      ],
      "name": "Query"
    }
  },
  "interfaces": {},
  "scalars": {
    "Float": {
      "name": "Float"
    },
    "String": {
      "name": "String"
    },
    "Boolean": {
      "name": "Boolean"
    },
    "Int": {
      "name": "Int"
    },
    "Url": {
      "name": "Url"
    }
  },
  "inputs": {}
}